source/local/local.go

```diff
@@ -32,6 +32,7 @@ import (
 // Config
 var (
 	hookDir = "/etc/kubernetes/node-feature-discovery/source.d/"
+	readDir = "/etc/kubernetes/node-feature-discovery/features.d/"
```

Comment:
I totally agree with you on this. I do not like the `reads` naming, but `files` doesn't sound better as hooks are also files and it's confusing between both while reading the code. I would like to have something that clearly marks the difference with the hooks and the `read` global meaning. `featureFilesDir` is better for the directory variable, but not for the rest of the code. We need to find something without `file` and `read` IMO.

Reply:
I've pushed a new version with a different naming. Tell me what do you think about it

Reply:
> featureFilesDir is better for the directory variable, but not for the rest of the code. We need to find something without file and read IMO.

I agree. Maybe we could still do some bike-shedding on this naming subject in order to get rid of **file** or **read** in the name.

Could something derived from **external** or **override** be better?


> I've pushed a new version with a different naming. Tell me what do you think about it

Ignoring this one naming issue, it looks good to me at a quick glance.

Reply:
I think we can keep the current naming. Despite the fact it can be confusing at some point (and it's better with the last modifications IMO), it has a sense. If we try to find something really different I'm afraid that it makes less sense.

Reply:
Let me sleep over it and see if my sub-consciousness magically generates a better suggestion 😄

Reply:
Ahah, sounds good to me :smile:

Reply:
I didn't come up with any better naming. The current naming is probably fine, especially as it's only internal details.